src/Joint.h

```diff
@@ -416,7 +462,15 @@ inline sva::MotionVecd Joint::tanAccel(const std::vector<double>& alphaD) const
 
 inline std::vector<double> Joint::zeroParam() const
 {
-	return ZeroParam(type_);
+	auto q = ZeroParam(type_);
+	if(is_mimic_)
+	{
+		for(auto & qi : q)
+		{
+			qi += mimic_offset_;
```

Comment:
What happens if we want to have a non-uniform mimic offset on a non-revolute joint ? mimic_offset should in that case probably be a `std::vector<double>`.